src/app/pages/app-bar/dropdown-toolbar/dropdown-toolbar.component.html

```diff
@@ -0,0 +1,17 @@
+<pxb-dropdown-toolbar #toolbar title="Title" [subtitle]="getSelected() || 'subtitle'">
+    <button
+        pxb-nav-icon
+        data-cy="toolbar-menu"
+        *ngIf="isSmall"
+        style="margin-left: -8px"
+        mat-icon-button
+        (click)="toggleMenu()"
+    >
+        <mat-icon>menu</mat-icon>
+    </button>
+    <ng-container pxb-toolbar-menu>
+        <ng-container *ngFor="let item of items">
+            <button mat-menu-item (click)="setSelected(item)">{{ item }}</button></ng-container
```

Comment:
Can we use list items instead of buttons here? Like what you had in `app-dropdown-toolbar-bottomsheet`

Reply:
I based this example off of what we're doing in the storybook demo.  Just to sanity check, ng material's menu demos also use buttons with the `mat-menu-item` directive applied.

Reply:
The design side has been using the more listy look.

(I personally don't mind, but just want to stay consistent)

Reply:
I'm still a little confused what makes the menu overlay look listy vs using buttons.

Reply:
Bottomsheet is updated to look more like a list; for the toolbar's menu, it has some top & bottom padding.  If this should be changed by default, then this sounds like a component-library change. 

![image](https://user-images.githubusercontent.com/6538289/133806646-69547ae7-7887-43e9-9085-e113e0a6df7c.png)